Budgett/utils/utils.js

```diff
@@ -0,0 +1,17 @@
+const fetch = require('node-fetch');
+const DomParser = require('dom-parser');
+const {wednesdayPath, nGagPath, defaultMeme} = require('../config.json');
+
+getWednesdayMeme = () =>{
+    return fetch(wednesdayPath)
+        .then(res => res.text())
+        .then(responseText => {
+            const doc = (new DomParser).parseFromString(responseText, 'text/html');
+            const arrMemes = doc.getElementById("jsid-latest-entries").textContent.split(',');
+            let rdmMeme = arrMemes[Math.floor(Math.random() * arrMemes.length)];
+            return nGagPath + rdmMeme;
+        })
+        .catch(() => {return defaultMeme})
+}
+
+module.exports = getWednesdayMeme;
```

Comment:
Do we plan on putting more modules in this file or are we planning to make a new file for each utility we want to build? If we're going to leave it as one utility per file, perhaps we should name this file to be more descriptive.

Reply:
Was planning on adding more modules as we move forward. Not sure what those would be, but I'm not too crazy about adding a new file per feature.